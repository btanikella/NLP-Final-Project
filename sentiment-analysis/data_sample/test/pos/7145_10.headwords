move
packed
gave
interesting
use
enjoyed
