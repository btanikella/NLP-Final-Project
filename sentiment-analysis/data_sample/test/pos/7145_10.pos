DT NN VBD RB VB PRP IN RB IN DT RBR  NNP NN VBG NNP NNP CC NNP NNP
IN PRP VBD RB VBN CC RB VBD  DT NNP NN VBD RBR VBN IN NN WDT VBD NN WDT PRP VBD JJ
PRP VBD PRP DT JJ NN IN DT JJ  JJ CC JJ NNS WDT VBD NNP NNP NNP
PRP VBD RB RBR JJ CC JJ IN DT NNP NN  CC RB RB IN RB JJ CC RB VBG
RB  IN PRP VBD RB VB DT JJ NN IN JJ NN IN DT NN  PRP VBD VB CD  CC IN JJ NNS
PRP VBD DT NN DT NN RB RB IN DT NNP NN  CC VBD JJ IN DT VBN NN PRP VBD
