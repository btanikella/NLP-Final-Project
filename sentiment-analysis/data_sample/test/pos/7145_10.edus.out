Sentence #1 (21 tokens):
This version did not move me as deeply as the later, Hollywood version starring Anthony Hopkins and Debra Winger.
[Text=This CharacterOffsetBegin=4 CharacterOffsetEnd=8 PartOfSpeech=DT] [Text=version CharacterOffsetBegin=9 CharacterOffsetEnd=16 PartOfSpeech=NN] [Text=did CharacterOffsetBegin=17 CharacterOffsetEnd=20 PartOfSpeech=VBD] [Text=not CharacterOffsetBegin=21 CharacterOffsetEnd=24 PartOfSpeech=RB] [Text=move CharacterOffsetBegin=25 CharacterOffsetEnd=29 PartOfSpeech=VB] [Text=me CharacterOffsetBegin=30 CharacterOffsetEnd=32 PartOfSpeech=PRP] [Text=as CharacterOffsetBegin=33 CharacterOffsetEnd=35 PartOfSpeech=IN] [Text=deeply CharacterOffsetBegin=36 CharacterOffsetEnd=42 PartOfSpeech=RB] [Text=as CharacterOffsetBegin=43 CharacterOffsetEnd=45 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=46 CharacterOffsetEnd=49 PartOfSpeech=DT] [Text=later CharacterOffsetBegin=50 CharacterOffsetEnd=55 PartOfSpeech=RBR] [Text=, CharacterOffsetBegin=55 CharacterOffsetEnd=56 PartOfSpeech=,] [Text=Hollywood CharacterOffsetBegin=57 CharacterOffsetEnd=66 PartOfSpeech=NNP] [Text=version CharacterOffsetBegin=67 CharacterOffsetEnd=74 PartOfSpeech=NN] [Text=starring CharacterOffsetBegin=75 CharacterOffsetEnd=83 PartOfSpeech=VBG] [Text=Anthony CharacterOffsetBegin=84 CharacterOffsetEnd=91 PartOfSpeech=NNP] [Text=Hopkins CharacterOffsetBegin=92 CharacterOffsetEnd=99 PartOfSpeech=NNP] [Text=and CharacterOffsetBegin=100 CharacterOffsetEnd=103 PartOfSpeech=CC] [Text=Debra CharacterOffsetBegin=104 CharacterOffsetEnd=109 PartOfSpeech=NNP] [Text=Winger CharacterOffsetBegin=110 CharacterOffsetEnd=116 PartOfSpeech=NNP] [Text=. CharacterOffsetBegin=116 CharacterOffsetEnd=117 PartOfSpeech=.] 
root(ROOT-0, move-5)
det(version-2, This-1)
nsubj(move-5, version-2)
aux(move-5, did-3)
neg(move-5, not-4)
dobj(move-5, me-6)
prep(move-5, as-7)
pcomp(as-7, deeply-8)
prep(deeply-8, as-9)
pobj(as-9, the-10)
dep(the-10, later-11)
punct(later-11, ,-12)
nn(version-14, Hollywood-13)
dep(,-12, version-14)
partmod(version-14, starring-15)
nn(Hopkins-17, Anthony-16)
dobj(starring-15, Hopkins-17)
cc(Hopkins-17, and-18)
nn(Winger-20, Debra-19)
conj(Hopkins-17, Winger-20)
punct(version-14, .-21)

Sentence #2 (25 tokens):
While it was beautifully filmed and superbly acted, the BBC version was more packed with dialog that imparted information which I found fascinating.
[Text=While CharacterOffsetBegin=122 CharacterOffsetEnd=127 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=128 CharacterOffsetEnd=130 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=131 CharacterOffsetEnd=134 PartOfSpeech=VBD] [Text=beautifully CharacterOffsetBegin=135 CharacterOffsetEnd=146 PartOfSpeech=RB] [Text=filmed CharacterOffsetBegin=147 CharacterOffsetEnd=153 PartOfSpeech=VBN] [Text=and CharacterOffsetBegin=154 CharacterOffsetEnd=157 PartOfSpeech=CC] [Text=superbly CharacterOffsetBegin=158 CharacterOffsetEnd=166 PartOfSpeech=RB] [Text=acted CharacterOffsetBegin=167 CharacterOffsetEnd=172 PartOfSpeech=VBD] [Text=, CharacterOffsetBegin=172 CharacterOffsetEnd=173 PartOfSpeech=,] [Text=the CharacterOffsetBegin=174 CharacterOffsetEnd=177 PartOfSpeech=DT] [Text=BBC CharacterOffsetBegin=178 CharacterOffsetEnd=181 PartOfSpeech=NNP] [Text=version CharacterOffsetBegin=182 CharacterOffsetEnd=189 PartOfSpeech=NN] [Text=was CharacterOffsetBegin=190 CharacterOffsetEnd=193 PartOfSpeech=VBD] [Text=more CharacterOffsetBegin=194 CharacterOffsetEnd=198 PartOfSpeech=RBR] [Text=packed CharacterOffsetBegin=199 CharacterOffsetEnd=205 PartOfSpeech=VBN] [Text=with CharacterOffsetBegin=206 CharacterOffsetEnd=210 PartOfSpeech=IN] [Text=dialog CharacterOffsetBegin=211 CharacterOffsetEnd=217 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=218 CharacterOffsetEnd=222 PartOfSpeech=WDT] [Text=imparted CharacterOffsetBegin=223 CharacterOffsetEnd=231 PartOfSpeech=VBD] [Text=information CharacterOffsetBegin=232 CharacterOffsetEnd=243 PartOfSpeech=NN] [Text=which CharacterOffsetBegin=244 CharacterOffsetEnd=249 PartOfSpeech=WDT] [Text=I CharacterOffsetBegin=250 CharacterOffsetEnd=251 PartOfSpeech=PRP] [Text=found CharacterOffsetBegin=252 CharacterOffsetEnd=257 PartOfSpeech=VBD] [Text=fascinating CharacterOffsetBegin=258 CharacterOffsetEnd=269 PartOfSpeech=JJ] [Text=. CharacterOffsetBegin=269 CharacterOffsetEnd=270 PartOfSpeech=.] 
root(ROOT-0, packed-15)
mark(filmed-5, While-1)
nsubj(filmed-5, it-2)
cop(filmed-5, was-3)
advmod(filmed-5, beautifully-4)
advcl(packed-15, filmed-5)
cc(filmed-5, and-6)
conj(filmed-5, superbly-7)
conj(filmed-5, acted-8)
punct(packed-15, ,-9)
det(version-12, the-10)
nn(version-12, BBC-11)
nsubjpass(packed-15, version-12)
auxpass(packed-15, was-13)
advmod(packed-15, more-14)
prep(packed-15, with-16)
pobj(with-16, dialog-17)
nsubj(imparted-19, that-18)
rcmod(dialog-17, imparted-19)
dobj(imparted-19, information-20)
dobj(found-23, which-21)
nsubj(found-23, I-22)
rcmod(information-20, found-23)
acomp(found-23, fascinating-24)
punct(packed-15, .-25)

Sentence #3 (20 tokens):
It gave me a detailed glimpse of the intellectual, theological and moral considerations that motivated C. S. Lewis.
[Text=It CharacterOffsetBegin=275 CharacterOffsetEnd=277 PartOfSpeech=PRP] [Text=gave CharacterOffsetBegin=278 CharacterOffsetEnd=282 PartOfSpeech=VBD] [Text=me CharacterOffsetBegin=283 CharacterOffsetEnd=285 PartOfSpeech=PRP] [Text=a CharacterOffsetBegin=286 CharacterOffsetEnd=287 PartOfSpeech=DT] [Text=detailed CharacterOffsetBegin=288 CharacterOffsetEnd=296 PartOfSpeech=JJ] [Text=glimpse CharacterOffsetBegin=297 CharacterOffsetEnd=304 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=305 CharacterOffsetEnd=307 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=308 CharacterOffsetEnd=311 PartOfSpeech=DT] [Text=intellectual CharacterOffsetBegin=312 CharacterOffsetEnd=324 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=324 CharacterOffsetEnd=325 PartOfSpeech=,] [Text=theological CharacterOffsetBegin=326 CharacterOffsetEnd=337 PartOfSpeech=JJ] [Text=and CharacterOffsetBegin=338 CharacterOffsetEnd=341 PartOfSpeech=CC] [Text=moral CharacterOffsetBegin=342 CharacterOffsetEnd=347 PartOfSpeech=JJ] [Text=considerations CharacterOffsetBegin=348 CharacterOffsetEnd=362 PartOfSpeech=NNS] [Text=that CharacterOffsetBegin=363 CharacterOffsetEnd=367 PartOfSpeech=WDT] [Text=motivated CharacterOffsetBegin=368 CharacterOffsetEnd=377 PartOfSpeech=VBD] [Text=C. CharacterOffsetBegin=378 CharacterOffsetEnd=380 PartOfSpeech=NNP] [Text=S. CharacterOffsetBegin=381 CharacterOffsetEnd=383 PartOfSpeech=NNP] [Text=Lewis CharacterOffsetBegin=384 CharacterOffsetEnd=389 PartOfSpeech=NNP] [Text=. CharacterOffsetBegin=389 CharacterOffsetEnd=390 PartOfSpeech=.] 
root(ROOT-0, gave-2)
nsubj(gave-2, It-1)
iobj(gave-2, me-3)
det(glimpse-6, a-4)
amod(glimpse-6, detailed-5)
dobj(gave-2, glimpse-6)
prep(glimpse-6, of-7)
det(theological-11, the-8)
amod(theological-11, intellectual-9)
punct(theological-11, ,-10)
pobj(of-7, theological-11)
cc(theological-11, and-12)
amod(considerations-14, moral-13)
conj(theological-11, considerations-14)
nsubj(motivated-16, that-15)
rcmod(theological-11, motivated-16)
nn(Lewis-19, C.-17)
nn(Lewis-19, S.-18)
dobj(motivated-16, Lewis-19)
punct(gave-2, .-20)

Sentence #4 (22 tokens):
It was therefore more interesting and stimulating than the Hollywood version, but not nearly as visually stunning or viscerally affecting.
[Text=It CharacterOffsetBegin=395 CharacterOffsetEnd=397 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=398 CharacterOffsetEnd=401 PartOfSpeech=VBD] [Text=therefore CharacterOffsetBegin=402 CharacterOffsetEnd=411 PartOfSpeech=RB] [Text=more CharacterOffsetBegin=412 CharacterOffsetEnd=416 PartOfSpeech=RBR] [Text=interesting CharacterOffsetBegin=417 CharacterOffsetEnd=428 PartOfSpeech=JJ] [Text=and CharacterOffsetBegin=429 CharacterOffsetEnd=432 PartOfSpeech=CC] [Text=stimulating CharacterOffsetBegin=433 CharacterOffsetEnd=444 PartOfSpeech=JJ] [Text=than CharacterOffsetBegin=445 CharacterOffsetEnd=449 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=450 CharacterOffsetEnd=453 PartOfSpeech=DT] [Text=Hollywood CharacterOffsetBegin=454 CharacterOffsetEnd=463 PartOfSpeech=NNP] [Text=version CharacterOffsetBegin=464 CharacterOffsetEnd=471 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=471 CharacterOffsetEnd=472 PartOfSpeech=,] [Text=but CharacterOffsetBegin=473 CharacterOffsetEnd=476 PartOfSpeech=CC] [Text=not CharacterOffsetBegin=477 CharacterOffsetEnd=480 PartOfSpeech=RB] [Text=nearly CharacterOffsetBegin=481 CharacterOffsetEnd=487 PartOfSpeech=RB] [Text=as CharacterOffsetBegin=488 CharacterOffsetEnd=490 PartOfSpeech=IN] [Text=visually CharacterOffsetBegin=491 CharacterOffsetEnd=499 PartOfSpeech=RB] [Text=stunning CharacterOffsetBegin=500 CharacterOffsetEnd=508 PartOfSpeech=JJ] [Text=or CharacterOffsetBegin=509 CharacterOffsetEnd=511 PartOfSpeech=CC] [Text=viscerally CharacterOffsetBegin=512 CharacterOffsetEnd=522 PartOfSpeech=RB] [Text=affecting CharacterOffsetBegin=523 CharacterOffsetEnd=532 PartOfSpeech=VBG] [Text=. CharacterOffsetBegin=532 CharacterOffsetEnd=533 PartOfSpeech=.] 
root(ROOT-0, interesting-5)
nsubj(interesting-5, It-1)
cop(interesting-5, was-2)
advmod(interesting-5, therefore-3)
advmod(interesting-5, more-4)
cc(interesting-5, and-6)
conj(interesting-5, stimulating-7)
prep(interesting-5, than-8)
det(version-11, the-9)
nn(version-11, Hollywood-10)
pobj(than-8, version-11)
punct(interesting-5, ,-12)
cc(interesting-5, but-13)
conj(interesting-5, not-14)
advmod(not-14, nearly-15)
prep(nearly-15, as-16)
advmod(stunning-18, visually-17)
amod(affecting-21, stunning-18)
cc(stunning-18, or-19)
conj(stunning-18, viscerally-20)
pcomp(as-16, affecting-21)
punct(interesting-5, .-22)

Sentence #5 (27 tokens):
Still, while I did not leave a heaping mound of sodden Kleenex in the theater, I did use one, and at frequent intervals.
[Text=Still CharacterOffsetBegin=538 CharacterOffsetEnd=543 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=543 CharacterOffsetEnd=544 PartOfSpeech=,] [Text=while CharacterOffsetBegin=545 CharacterOffsetEnd=550 PartOfSpeech=IN] [Text=I CharacterOffsetBegin=551 CharacterOffsetEnd=552 PartOfSpeech=PRP] [Text=did CharacterOffsetBegin=553 CharacterOffsetEnd=556 PartOfSpeech=VBD] [Text=not CharacterOffsetBegin=557 CharacterOffsetEnd=560 PartOfSpeech=RB] [Text=leave CharacterOffsetBegin=561 CharacterOffsetEnd=566 PartOfSpeech=VB] [Text=a CharacterOffsetBegin=567 CharacterOffsetEnd=568 PartOfSpeech=DT] [Text=heaping CharacterOffsetBegin=569 CharacterOffsetEnd=576 PartOfSpeech=JJ] [Text=mound CharacterOffsetBegin=577 CharacterOffsetEnd=582 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=583 CharacterOffsetEnd=585 PartOfSpeech=IN] [Text=sodden CharacterOffsetBegin=586 CharacterOffsetEnd=592 PartOfSpeech=JJ] [Text=Kleenex CharacterOffsetBegin=593 CharacterOffsetEnd=600 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=601 CharacterOffsetEnd=603 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=604 CharacterOffsetEnd=607 PartOfSpeech=DT] [Text=theater CharacterOffsetBegin=608 CharacterOffsetEnd=615 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=615 CharacterOffsetEnd=616 PartOfSpeech=,] [Text=I CharacterOffsetBegin=617 CharacterOffsetEnd=618 PartOfSpeech=PRP] [Text=did CharacterOffsetBegin=619 CharacterOffsetEnd=622 PartOfSpeech=VBD] [Text=use CharacterOffsetBegin=623 CharacterOffsetEnd=626 PartOfSpeech=VB] [Text=one CharacterOffsetBegin=627 CharacterOffsetEnd=630 PartOfSpeech=CD] [Text=, CharacterOffsetBegin=630 CharacterOffsetEnd=631 PartOfSpeech=,] [Text=and CharacterOffsetBegin=632 CharacterOffsetEnd=635 PartOfSpeech=CC] [Text=at CharacterOffsetBegin=636 CharacterOffsetEnd=638 PartOfSpeech=IN] [Text=frequent CharacterOffsetBegin=639 CharacterOffsetEnd=647 PartOfSpeech=JJ] [Text=intervals CharacterOffsetBegin=648 CharacterOffsetEnd=657 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=657 CharacterOffsetEnd=658 PartOfSpeech=.] 
root(ROOT-0, use-20)
advmod(use-20, Still-1)
punct(use-20, ,-2)
mark(leave-7, while-3)
nsubj(leave-7, I-4)
aux(leave-7, did-5)
neg(leave-7, not-6)
advcl(use-20, leave-7)
det(mound-10, a-8)
amod(mound-10, heaping-9)
dobj(leave-7, mound-10)
prep(mound-10, of-11)
amod(Kleenex-13, sodden-12)
pobj(of-11, Kleenex-13)
prep(Kleenex-13, in-14)
det(theater-16, the-15)
pobj(in-14, theater-16)
punct(use-20, ,-17)
nsubj(use-20, I-18)
aux(use-20, did-19)
dobj(use-20, one-21)
punct(use-20, ,-22)
cc(use-20, and-23)
conj(use-20, at-24)
amod(intervals-26, frequent-25)
pobj(at-24, intervals-26)
punct(use-20, .-27)

Sentence #6 (23 tokens):
I enjoyed this film every bit as much as the Hollywood version, and was grateful for the increased understanding I gained.
[Text=I CharacterOffsetBegin=663 CharacterOffsetEnd=664 PartOfSpeech=PRP] [Text=enjoyed CharacterOffsetBegin=665 CharacterOffsetEnd=672 PartOfSpeech=VBD] [Text=this CharacterOffsetBegin=673 CharacterOffsetEnd=677 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=678 CharacterOffsetEnd=682 PartOfSpeech=NN] [Text=every CharacterOffsetBegin=683 CharacterOffsetEnd=688 PartOfSpeech=DT] [Text=bit CharacterOffsetBegin=689 CharacterOffsetEnd=692 PartOfSpeech=NN] [Text=as CharacterOffsetBegin=693 CharacterOffsetEnd=695 PartOfSpeech=RB] [Text=much CharacterOffsetBegin=696 CharacterOffsetEnd=700 PartOfSpeech=RB] [Text=as CharacterOffsetBegin=701 CharacterOffsetEnd=703 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=704 CharacterOffsetEnd=707 PartOfSpeech=DT] [Text=Hollywood CharacterOffsetBegin=708 CharacterOffsetEnd=717 PartOfSpeech=NNP] [Text=version CharacterOffsetBegin=718 CharacterOffsetEnd=725 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=725 CharacterOffsetEnd=726 PartOfSpeech=,] [Text=and CharacterOffsetBegin=727 CharacterOffsetEnd=730 PartOfSpeech=CC] [Text=was CharacterOffsetBegin=731 CharacterOffsetEnd=734 PartOfSpeech=VBD] [Text=grateful CharacterOffsetBegin=735 CharacterOffsetEnd=743 PartOfSpeech=JJ] [Text=for CharacterOffsetBegin=744 CharacterOffsetEnd=747 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=748 CharacterOffsetEnd=751 PartOfSpeech=DT] [Text=increased CharacterOffsetBegin=752 CharacterOffsetEnd=761 PartOfSpeech=VBN] [Text=understanding CharacterOffsetBegin=762 CharacterOffsetEnd=775 PartOfSpeech=NN] [Text=I CharacterOffsetBegin=776 CharacterOffsetEnd=777 PartOfSpeech=PRP] [Text=gained CharacterOffsetBegin=778 CharacterOffsetEnd=784 PartOfSpeech=VBD] [Text=. CharacterOffsetBegin=784 CharacterOffsetEnd=785 PartOfSpeech=.] 
root(ROOT-0, enjoyed-2)
nsubj(enjoyed-2, I-1)
det(film-4, this-3)
dobj(enjoyed-2, film-4)
det(bit-6, every-5)
tmod(enjoyed-2, bit-6)
advmod(much-8, as-7)
amod(bit-6, much-8)
prep(much-8, as-9)
det(version-12, the-10)
nn(version-12, Hollywood-11)
pobj(as-9, version-12)
punct(enjoyed-2, ,-13)
cc(enjoyed-2, and-14)
cop(grateful-16, was-15)
conj(enjoyed-2, grateful-16)
prep(grateful-16, for-17)
det(understanding-20, the-18)
amod(understanding-20, increased-19)
pobj(for-17, understanding-20)
nsubj(gained-22, I-21)
rcmod(understanding-20, gained-22)
punct(enjoyed-2, .-23)

